### ISSUE TITLE
Add `id` field to notebook YAML

### ISSUE THREAD

**Issue by ckadner** [machine-learning-exchange/katalog#35]
Currently the notebook YAMLs do not have an `id` or `notebook_identifier` field. This makes it hard to reference a notebook as a `related_asset` from other assets, like datasets.

**Comment by ckadner** [comment]
@JAulet

**Comment by JAulet** [comment]
So for this issue I should go through the notebook YAMLs in this repository and add the id? @ckadner

**Comment by ckadner** [comment]
Yes. For example, the [CodeNet Language Classification notebook](https://github.com/machine-learning-exchange/katalog/blob/main/model-samples/codenet-language-classification/codenet-language-classification.yaml) would get this id:

```YAML
id: codenet-langclass
```
